x-pack/plugins/apm/server/routes/errors/distribution/get_buckets.ts

```diff
@@ -36,7 +37,12 @@ export async function getBuckets({
 }) {
   const params = {
     apm: {
-      events: [ProcessorEvent.error],
```

Comment:
I wonder if we should deprecate the `events`?

Reply:
That's what we are working on, I'm changing all the APIs to use source, I left some for follow-ups because are more tricky to change, there are some queries that check several indices for example